content/source/docs/enterprise/api/workspaces.html.md

```diff
@@ -213,10 +213,12 @@ Update the workspace settings
 
 ### Parameters
 
+Note that only the first four of these parameters can be used when updating a workspace without a VCS repository.
```

Comment:
I'd probably make this something like "Workspaces without an associated VCS repository only use the organization name, workspace name, terraform version, and working directory."